packages/ui/src/components/molecules/Alert/Alert.tsx

```diff
@@ -133,7 +135,7 @@ const Alert = ({
           {children}
           {(isCancelButtonDisplayedBottom ||
             isConfirmButtonDisplayedBottom) && (
-            <View style={styles.buttonsContainer}>
+            <View style={[styles.buttonsContainer, buttonsContainerStyle]}>
```

Comment:
This is not used in this PR, please create a new PR on dev branch to add the style and remove the change here